src/tcgwars/logic/impl/gen8/DarknessAblaze.groovy

```diff
@@ -1051,6 +1054,16 @@ public enum DarknessAblaze implements LogicCardInfo {
         bwAbility "Shuffle Dance", {
           text "Once during your turn, you may choose 1 of your opponent’s face-down Prize cards and switch it with the top card from their deck. (Both cards remain face-down.)"
           actionA {
+            checkLastTurn()
+            assert opp.deck : "Your opponent's deck is empty"
+            powerUsed()
+
+            def info = "Select opponent's Prize card to switch with the top card from their deck."
```

Comment:
Just a comment but liking this info variable, might steal the idea on future implementations I make :P

Reply:
Anything to keep lines from reaching the 100 char marker and wrapping in my VSCode :laughing: